rpm/packetfence.spec

```diff
@@ -1038,6 +1038,8 @@ fi
 %dir                    /usr/local/pf/conf/I18N
 %dir                    /usr/local/pf/conf/I18N/api
                         /usr/local/pf/conf/I18N/api/*
+%dir                    /usr/local/pf/conf/kafka
+%config                 /usr/local/pf/conf/kafka/kafka_server_jaas.conf.tt
 %dir                    /usr/local/pf/conf/locale
```

Comment:
You should add kafka.conf too as a config file (when we do an upgrade it will override it)

Reply:
Also in debian/packetfence.conffiles